src/renderers/webgpu/WebGPUBackend.js

```diff
@@ -955,7 +955,7 @@ class WebGPUBackend extends Backend {
 			const drawCount = object._multiDrawCount;
 			const drawInstances = object._multiDrawInstances;
 
-			const bytesPerElement = index.bytesPerElement || 1;
+			const bytesPerElement = index ? index.bytesPerElement : 1;
```

Comment:
`index !== undefined`, `index !== null`, `index !== 0`, ... ?

Reply:
if index === undifned or  index === null, "bytesPerElement" will be 1 and because  this code 
`passEncoderGPU.drawIndexed( counts[ i ] / bytesPerElement, count, starts[ i ] / 4, 0, firstInstance );`
"bytesPerElement" must not be 0

Reply:
At first I was worried that it might be undefined，then I double check bufferGeometry.js. It default value is null，so your suggestion is ok